mkdocs/commands/build.py

```diff
@@ -339,14 +359,14 @@ def build(config, live_server=False, dump_json=False, clean_site_dir=False):
     for theme_dir in reversed(config['theme_dir']):
         log.debug("Copying static assets from theme: %s", theme_dir)
         utils.copy_media_files(
-            theme_dir, config['site_dir'], exclude=['*.py', '*.pyc', '*.html']
+            theme_dir, config['site_dir'], exclude=['*.py', '*.pyc', '*.html'], dirty=dirty
         )
 
     log.debug("Copying static assets from the docs dir.")
     utils.copy_media_files(config['docs_dir'], config['site_dir'])
```

Comment:
Don't we want to pass the value of `dirty` on here? Same applies to any other calls to this function or `utils.copy_file`.